### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by mohitp333**
Added built script for empty

### CODE REVIEW

**Comment by seth-priya** [comment]
@Ryan-Dmello please review

File: e/empty/empty_rhel_7.6.sh.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        nvm install v10.24.0
+fi
+
+nvm alias default v10.24.0
+
+# Install and test empty
+git clone https://github.com/iclanzan/empty.git
+cd empty/
+#git checkout ${PACKAGE_VERSION}
+# npm install -g yarn
```
Comment (Ryan-Dmello) [inline e/empty/empty_rhel_7.6.sh.txt:43]: Please remove commented lines

File: e/empty/empty_rhel_7.6.sh.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Version       : 0.10.1
+# Source        : https://github.com/iclanzan/empty.git
+# Tested on     : RHEL 8.3
+# Node Version  : v10.24.0
```
Comment (Ryan-Dmello) [inline e/empty/empty_rhel_7.6.sh.txt:4]: Please mention package version also

File: e/empty/empty_rhel_7.6.sh.txt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+set -e
+
+# Install all dependencies.
+yum clean all
+yum -y update
```
Comment (Ryan-Dmello) [inline e/empty/empty_rhel_7.6.sh.txt:18]: if installation is required(i.e root), please mention that in script documentation

**Comment by Ryan-Dmello** [comment]
File naming convention is not matching with distro its verified on(as per doc string in the script). Please change the script name to empty_rhel_8.3.sh(if tested on RHEL 8.3)

**Review by Ryan-Dmello — CHANGES_REQUESTED** [review]

**Comment by seth-priya** [comment]
@mohitp333 any update on this?

**Comment by seth-priya** [comment]
@Ryan-Dmello please review

**Comment by Ryan-Dmello** [comment]
@seth-priya, the previous review comments are still not incorporated/justified yet. I have no new review comments other than the once i already mentioned above

**Comment by seth-priya** [comment]
@mohitp333 any update on this?

**Comment by mohitp333** [comment]
Hi Priya,
 
Sorry for delay in update can I get sometime, I am just verifying it once.
----- Original message -----From: "Priya Seth" ***@***.***>To: "ppc64le/build-scripts" ***@***.***>Cc: "mohitp333" ***@***.***>, "Mention" ***@***.***>Subject: [EXTERNAL] Re: [ppc64le/build-scripts] Add files via upload (#856)Date: Fri, Oct 8, 2021 10:52 AM      @mohitp333 any update on this? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. ‍ ‍ ‍ ‍ ‍ ‍ ‍ ‍ ‍ ‍ ‍ ‍ ‍ ‍    ZjQcmQRYFpfptBannerStart  

This Message Is From an External Sender
This message came from outside your organization. ZjQcmQRYFpfptBannerEnd 
 
@mohitp333 any update on this?

**Comment by seth-priya** [comment]
@Ryan-Dmello please review and confirm

**Comment by Ryan-Dmello** [comment]
Doc string is missing Package name section. Also the package version is set to master. @mohitp333 Please verify and make changes as required

**Comment by Ryan-Dmello** [comment]
Changes look ok

**Pull request merged**